.github/ISSUE_TEMPLATE/ADOPTERS.yaml

```diff
@@ -0,0 +1,54 @@
+name: Let us know you are using Knative
+description: If your company is using Knative in production or some stage of development, please add your company information here and it will be added to the ADOPTERS.md file.
+title: "[ADOPTERS]: ${COMPANY NAME HERE}"
```

Comment:
@lance Can we use something else than "COMPANY" it could be another open source project integrating or depending on knative. How about "ORGANIZATION" ?

Reply:
@csantanapr want to make a PR?